src/application/clients/crux-wallet-client.ts

```diff
@@ -121,15 +124,34 @@ export class CruxWalletClient {
         if (!cruxUser) {
             throw ErrorHelper.getPackageError(null, PackageErrorCode.UserDoesNotExist);
         }
-        const assetId = this.getCruxAssetTranslator().symbolToAssetId(walletCurrencySymbol);
-        if (!assetId) {
-            throw ErrorHelper.getPackageError(null, PackageErrorCode.AssetIDNotAvailable);
+        const addressResolver = new CruxAddressResolver({
+            cruxAssetTranslator: this.getCruxAssetTranslator(),
+            cruxUser,
+        });
+        const userAddress = addressResolver.resolveAddressBySymbol(walletCurrencySymbol);
+        return userAddress;
+    }
+
+    @throwCruxClientError
+    public resolveAssetAddressForCruxID = async (fullCruxID: string, assetMatcher: IAssetMatcher): Promise<IAddress> => {
+        await this.initPromise;
+        const tag = "resolving_address";
```

Comment:
Can we use different tags to identify the number of users actually using this new feature while resolving?